tests/run_tests.py

```diff
@@ -46,6 +46,8 @@ def main():
                 path
             )
         )
+        library_archive, _ = os.path.splitext(os.path.basename(module))
+        library_archive = os.path.join(library_archive, ".a")
```

Comment:
I don't think this is correct, this will give you `{library_archive}/.a`, I think you just want `library_archive + ".a"`?

Reply:
Ack, good catch!